code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -18,7 +24,43 @@
 /obj/item/clothing/mask/muzzle/attack_hand(mob/user as mob)
 	if(user.wear_mask == src && !user.IsAdvancedToolUser())
 		return 0
+	else if(user.wear_mask == src && security_lock && locked)
+		if(ishuman(user))
+			if(doUnlock(user))
+				visible_message("<span class='danger'>[user] unlocks their [src.name].</span>", \
+									"<span class='userdanger'>[user] unlocks their [src.name].</span>")
+		else
+			to_chat(user, "<span class='warning'>You lack the ability to open the lock.</span>")

```

Comment:
As far as I know, there is no possible way for this else branch to be called outside of admin abuse. Aliens are stopped at the `IsAdvancedToolUser` check, and aliens that _are_ advanced tool users _should_ be able to use this. If the intent was to stop monkeys from doing it, you need to check `issmall`, not `ishuman`.